Modules/III-k6-Intermediate/08-Setting-load-profiles-with-executors/Externally-Controlled-Exercises.md

```diff
@@ -126,9 +126,9 @@ curl -X PATCH \
 ### Script options
 Our initial script provides the bare minimum to begin your test. 
 
-Consider a best-practice of specifying your `maxVUs` within the script. As noted previously, the `maxVUs` is not required, however any attempt to scale will be met with an error unless a `--max` is specified with the initial scaling request. The `maxVUs` value may be overridden using the `--max` argument should the controlling script deem necessary.
+Consider specifying your `maxVUs` within the script as best practice. As noted previously, the `maxVUs` is not required. However, any attempt to scale will be met with an error unless a `--max` is specified with the initial scaling request. The `maxVUs` value may be overridden using the `--max` argument if deemed necessary.
```

Comment:
```suggestion
Consider specifying your `maxVUs` . As noted previously, the `maxVUs` is not required. However, any attempt to scale will be met with an error unless a `--max` is specified with the initial scaling request. The `maxVUs` value may be overridden using the `--max` argument if deemed necessary.
```

Reply:
I think we need to double check this info. It might not be even a recommended practice. But this is a question for the larger work of https://github.com/grafana/k6-docs/issues/808, not a blocking review.